README.md

```diff
@@ -1,3 +1,29 @@
-# dedx web app
+# dEdx Web
 
+An online interface of the dEdx library.
+## Live-dev version
 Development version available at https://aptg.github.io/web_dev/
+
+## General idea
+The idea of this project is to prepare a web interface of the dEdx library. Thanks to WebAssembly technology as well as clean and simple UI it will allow users to calculate and view stopping powers and energies of various particles.
+
+Part of the project is similar to the PSTAR program of the National Institue of Standards and Technology, but we aim to add more options to the queries, make the plots dynamicly generated, create a fluent user interface and - by focusing on responsivenes of the web layout - enable researchers from around the world to gather the necessery data on their mobile phones in lab enviroment.
+
+## Technologies used in the project
```

Comment:
I'd completely skip technologies in the main readme and move it to separate file (i.e. `technical.md`) and just add a link here